src/components/RequestsFilters/RequestsFilters.js

```diff
@@ -107,6 +110,13 @@ export default class RequestsFilters extends React.Component {
             ariaLabelledBy={requestFilterTypes.TAGS}
           />
         </Accordion>
+
+        <PickupServicePointFilter
+          activeValues={pickupServicePoints}
+          servicePoints={this.props.resources?.servicePoints?.records}
```

Comment:
IIRC, now `this.props.resources.servicePoints.records` always exists. By default it's `[]`